elastic/beats
# [Metricbeat] Add collecting AuroraDB metrics in rds metricset

## What does this PR do?

This PR is to add collecting AuroraDB metrics into rds metricset: https://docs.aws.amazon.com/AmazonRDS/latest/AuroraUserGuide/Aurora.Monitoring.html

## How to test this PR locally
1. create an AuroraDB service in AWS
2. enable aws module
3. change aws.yml config to:
```
- module: aws
  period: 5m
  credential_profile_name: elastic-test
  metricsets:
    - rds
```
4. You should see rds metrics from AuroraDB like:
`aws.rds.aurora_bin_log_replica_lag`, `aws.rds.aurora_replica.lag.ms`, `aurora_replica.lag_max.ms` and etc.

## Related issues
- Closes https://github.com/elastic/beats/issues/14142
